menu/menu.yaml

```diff
@@ -61,7 +61,7 @@
       Articles: 
         - "nservicebus/errors/critical-exception-for-timeout-outages"
         - "nservicebus/errors/exception-caveats"
-        - "nservicebus/errors/second-level-retries"
+        - "nservicebus/errors/automatic-retries"
```

Comment:
Just curious - are we going to call replace SLR name with "automatic retries" moving forward?